src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin: 0;
```

Comment:
It's great that you've addressed the last child with `&:last-child`, but the margin should be explicitly set to `margin-right: 0;` to override the right margin, not just `margin: 0;` which may lead to confusion.